addons/web_editor/static/src/scss/wysiwyg_snippets.scss

```diff
@@ -1674,22 +1674,22 @@
                 padding-right: .5em;
                 font-family: FontAwesome;
             }
+        }
 
-            input {
-                flex: 1 1 auto;
-                color: inherit;
-                border: $o-we-sidebar-content-field-border-width solid $o-we-sidebar-content-field-border-color;
-                border-radius: $o-we-sidebar-content-field-border-radius;
-                background-color: $o-we-sidebar-content-field-input-bg;
-                padding: 1px $o-we-sidebar-content-field-clickable-spacing;
+        :where(.o_we_m2o_search, .o_we_m2o_create) input {
+            flex: 1 1 auto;
+            color: inherit;
+            border: $o-we-sidebar-content-field-border-width solid $o-we-sidebar-content-field-border-color;
+            border-radius: $o-we-sidebar-content-field-border-radius;
+            background-color: $o-we-sidebar-content-field-input-bg;
+            padding: 1px $o-we-sidebar-content-field-clickable-spacing;
```

Comment:
I'm curious, why is this change needed?

Reply:
The `.o_we_m2o_create input` was a widget (so actually a `we-input`) and is now just a plain input, so the styles didn't match.
Although maybe I shouldn't use `:where()` to keep the same specificity as before for the `.o_we_m2o_search`.